src/serlo-editor/plugins/multimedia/renderer.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx'
+
+export interface MultimediaRendererProps {
+  mediaWidth: number // 25 | 50 | 75 | 100 Percent
```

Comment:
Is this still relevant if the user can only set two sizes in the settings?

Reply:
unfortunately yes, since this does not change currently stored values.